cmake/FindCoreAPI.cmake

```diff
@@ -86,7 +88,7 @@ function(CORE_API_GENERATE CORE_API_DIR SCRIPTS_DIR)
 
     list(TRANSFORM AOS_PROTO_SRC PREPEND ${CORE_API_DIR}/)
 
-    nanopb_generate_cpp(proto_sources proto_headers RELPATH ${CORE_API_DIR} ${AOS_PROTO_SRC})
+    nanopb_generate_cpp(proto_sources proto_headers RELPATH ${CORE_API_DIR}/proto ${AOS_PROTO_SRC})
```

Comment:
Instead of changing RELPATH please add the following at the top of this function:
```
    set(NANOPB_IMPORT_DIRS ${CORE_API_DIR}/proto)
```